### PR TITLE
t3 and t4

### PR DESCRIPTION
**Pull request by ZiaurRehmanmalhi**

### CODE REVIEW

File: task/t3.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+
+My name is zia ur rehman i am using vim editor i learn vim editor this is a small task i want this  file on pull request.
+vim editor is a terminal editor it is very healthfull for programers. in the vim editor make a file and type this massage save command is ":w" and go to back terminal command is ":q" and ":qw" comand both work
```
Comment (Hamma111) [inline task/t3.txt]: helpful* for programmers

File: task/t4.txt [hunk 2]
```diff
@@ -0,0 +1,5 @@
+this is my nano file.
+nano editor like a vim editor this is a simple editor 
+nano and vim editor easy to use. 
+this is vry helpfull editor.
+control x is save changes.
```
Comment (Hamma111) [inline task/t4.txt]: to save the file, we have to do one more thing after Ctrl+x, what is that?

**Review by Hamma111 — CHANGES_REQUESTED** [review]

File: task/t3.txt (hunk 1 above)
Comment (ZiaurRehmanmalhi) [reply task/t3.txt]: ok i will change

File: task/t4.txt (hunk 2 above)
Comment (ZiaurRehmanmalhi) [reply task/t4.txt]: has changed

File: task/t4.txt.save [hunk 3]
```diff
@@ -0,0 +1,5 @@
+this is my nano file.
```
Comment (Hamma111) [inline task/t4.txt.save]: why is this extra file here

**Review by Hamma111 — CHANGES_REQUESTED** [review]

File: task/t4.txt.save (hunk 3 above)
Comment (ZiaurRehmanmalhi) [reply task/t4.txt.save]: ye khud hi a gi delete krta hn

**Pull request merged**